float64 x
float64 y
string coord_sys
float64 dx
float64 dy
float64 dt
float64 num
float64 rampt
float64 delay
float64 off_x
float64 off_y
string offcoord
int64 dcos
string hosei
float64 lamda
bool limit
string from_node
float64 timestamp